data-final-fixes.lua

```diff
@@ -78,9 +78,11 @@ if settings.startup["py-braided-pipes"].value then
         ::continue::
     end
 
-    for _, pipe in pairs {data.raw.pipe.pipe, data.raw["pipe-to-ground"]["pipe-to-ground"]} do
+    for _, pipe in pairs {table.unpack(data.raw["pipe"]), table.unpack(data.raw["pipe-to-ground"])} do
         for _, pipe_connection in pairs(pipe.fluid_box.pipe_connections) do
-            pipe_connection.connection_category = {"pipe"}
+            if pipe_connection.connection_category == nil then
```

Comment:
I guess in theory there's a possible regression here: If some mod set the `connection_category` of base pipes/undergrounds to something not-nil and not `{"pipe"}`, then we're no longer un-doing that change here.

I'm inclined to think that in such a case pyindustry would have been _breaking_ that other mod somehow, and they should be either considered incompatible, or the other mod needs to be made py-aware